Helper/Data.php

```diff
@@ -643,23 +651,17 @@ public function isDemoMode($storeId = null)
     public function getAPIKey($storeId = null)
     {
         if ($this->isDemoMode($storeId)) {
-            $apiKey = $this->_encryptor->decrypt(
-                trim(
-                    $this->getAdyenAbstractConfigData(
-                        'api_key_test',
-                        $storeId
-                    )
-                )
-            );
+            $encryptedApiKeyTest = $this->getAdyenAbstractConfigData('api_key_test', $storeId);
+            if (is_null($encryptedApiKeyTest)) {
+                return false;
+            }
+            $apiKey = $this->_encryptor->decrypt(trim($encryptedApiKeyTest));
         } else {
-            $apiKey = $this->_encryptor->decrypt(
-                trim(
-                    $this->getAdyenAbstractConfigData(
-                        'api_key_live',
-                        $storeId
-                    )
-                )
-            );
+            $encryptedApiKeyLive = $this->getAdyenAbstractConfigData('api_key_live', $storeId);
+            if (is_null($encryptedApiKeyLive)) {
+                return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
                return null;
```